### ISSUE TITLE
Cannot connect to runtime process

### ISSUE THREAD

**Issue by pannous** [Microsoft/vscode#19738]
- VSCode Version: Code 1.9.0 (27240e71ef390bf2d66307e677c2a333cebf75af, 2017-02-02T08:23:30.856Z)
- OS Version: Darwin x64 16.4.0
- Extensions:

|Extension|Author|Version|
|---|---|---|
|TypeScriptImport|kevinmcgowan|1.7.0|
|vscode-exec-node|miramac|0.2.1|
|PowerShell|ms-vscode|0.9.0|
|csharp|ms-vscode|1.6.2|

---

Steps to Reproduce:

1. open a single test.js file, content:
x=1
console.log(x)
x=x+1
console.log(x)

2. press ctrl+r
DEBUG CONSOLE output:
```
node --debug-brk=29725 --nolazy test.js 
Debugger listening on 127.0.0.1:29725
```
Nothing happens. After half a minute:
Cannot connect to runtime process  (reason: connect ETIMEDOUT 127.0.53.53:29725).

Expected behaviour:
test.js is run, 
variable is printed
debugger stops at breakpoint (line 3)

Likely culprit : somehow localhost is resolved as 127.0.53.53 instead of 127.0.0.1 !?!?

**Comment by pannous** [comment]
Problem has been solved by adding 127.0.0.1	localhost.fritz.box to /etc/hosts

**Comment by Sacro** [comment]
Just dropping by to point out that 127.0.53.53 is the correct DNS response, it's to warn you that you're using someone else's domain, in this case *.box might be used by someone else.

https://www.icann.org/news/announcement-2-2014-08-01-en
